src/media-groups.js

```diff
@@ -250,13 +250,10 @@ export const onError = {
    */
   AUDIO: (type, settings) => () => {
     const {
-      segmentLoaders: { [type]: segmentLoader},
       mediaTypes: { [type]: mediaType },
       excludePlaylist
     } = settings;
 
-    stopLoaders(segmentLoader, mediaType);
```

Comment:
trying to understand why this has been removed?

Reply:
I believe we removed this here because this was a redundant call of `stopLoaders`. Apparently, stop loaders will always be called [here](https://github.com/videojs/http-streaming/blob/04451d4d44d2c9ac803b91898497225f2a56ea99/src/media-groups.js#L170) in this case, and then that function will be returned before calling `startLoaders` again..

Reply:
Does it mean that it should be removed for all other media-group types?

Reply:
I think you're right! I just removed the same piece of code from `SUBTITLES` here: https://github.com/videojs/http-streaming/pull/1427/commits/65f1016df5036783a457dda4ef068da7d0fa0bf8

Before resolving, we'll make sure that @adrums86 takes a look at this change and agrees.

Reply:
Can we add TODO, so we will not forget about it?

Reply:
So I removed `stopLoaders` from the other media group types in the `onError` function. It is still used in `onTrackChanged` and `onGroupChanged` where I believe it is still needed.